docs/usage/export.md

```diff
@@ -1,5 +1,57 @@
-Exporting data
+Exporting Data
 ==============
 
 ## Introduction
-Feature coming soon.
\ No newline at end of file
+
+The Bulk Export feature allows you to export data from a Sleeper table to an S3 bucket in a number of Parquet files.
+
+## Prerequisites
+Before performing a Bulk Export, ensure that the `BulkExportStack` has been deployed. This stack is optional and is not included in a vanilla installation.
+
+## How to Perform a Bulk Export
+
+### Step 1: Submit a Bulk Export Query
+
+To initiate a Bulk Export, submit a **Bulk Export Query** to the SQS queue specified in the `BULK_EXPORT_QUEUE_URL` property. The query should specify the Sleeper table name or table id to be exported. If wanted an `exportId` can also be provided, if one isn't then one is generated.
+
+The system will automatically split the query into smaller export tasks (Leaf Partition Bulk Export Queries) and process them.
+
+#### Example Query Message:
+```json
+{
+  "tableName": "example-table",
+  "exportId": "export-123"
+}
+```
+
+### Step 2: Monitor the Export Process
+
+Once the export query is submitted, the system will:
+1. Automatically split the Bulk Export Query into smaller export queries for each Leaf Partition.
+2. Add the Leaf Partition Bulk Export Queries to the SQS queue `LEAF_PARTITION_BULK_EXPORT_QUEUE_URL`.
+3. The ECS Bulk Export Task Runner will:
```

Comment:
I don't think bulk export task runner should be capitalised, as it's not a proper noun.